studio/components/interfaces/Settings/Logs/LogTable.tsx

```diff
@@ -44,10 +62,16 @@ const LogTable = ({
 
   const DEFAULT_COLUMNS = columnNames.map((v) => {
     let formatter = undefined
-    const firstRow: any = data[0]
-    const value = firstRow?.[v]
-    if (typeof value === 'object') {
-      formatter = () => `[Object]`
+
+    formatter = (received: FormatterArg) => {
+      const value = received.row?.[v]
+      if (value && typeof value === 'object') {
```

Comment:
do we not want to JSON.stringify here? this will handle `null` values too

Reply:
Hmm depends on whether we want to let the stringified object overflow in the table, which i think is a little redundant since they need to click the row to see the full object anyway.

As for handling individual values, using `JSON.stringify` on  a string will wrap it in quotes, which is not desirable for rendering.